ner_v1/detectors/pattern/pnr/pnr_detection.py

```diff
@@ -159,6 +177,42 @@ def _detect_railway_pnr_format(self, railway_pnr_list=None, original_list=None):
             original_list.append(pattern)
         return railway_pnr_list, original_list
 
+    def _detect_railway_pnr_clean_format(self, railway_pnr_list=[], original_list=[]):
```

Comment:
rename to `_detect_railway_pnr_long_format`

Reply:
Remove mutable arguments. 
`railway_pnr_list=[], original_list=[]` -> `railway_pnr_list=None, original_list=None`

and in the body initialize them to [] by checking them against None.
```python
        if railway_pnr_list is None:
            railway_pnr_list = []
```

Pycharm provides a shortcut to do it.